src/cclib/parser/orcaparser.py

```diff
@@ -151,7 +151,7 @@ def extract(self, inputfile, line):
             if not hasattr(self, "scftargets"):
                 self.scftargets = []
 
-            energy = self.scfvalues[-1][-1][0]
```

Comment:
I don't think there's a good answer here. I would say that as long as it's documented and clear what the units are, it does not matter too much. Since scfenergies/scfvalues are somehow "raw" (depend on the program), perhaps it is better not to convert the numbers.